pom.xml

```diff
@@ -23,6 +23,7 @@
 		<beanutils>1.9.4</beanutils>
 		<xstream>1.4.21</xstream>
 		<pool>2.11.1</pool>
+		<valkey.glide>2.1.1</valkey.glide>
```

Comment:
We'll update this to 2.2.0 when it is release in the next month or so in order to get Windows support.